### PR TITLE
Create cost details toolbar component with filter, sort, and export button.

### PR DESCRIPTION
**Pull request by chambridge**
* The intent is to setup the basic pieces within the page from the PF3 library
* Next step is to wire up the interactions of the toolbar with the filtering/ordering/export functions

image

### CODE REVIEW

**Comment by dmiller9911** [comment]
Deploy preview for *koku* ready!

Built with commit 5ee603c29815e1e26a0531191e2ed56dc714256e

https://deploy-preview-101--koku.netlify.com

**Comment by dlabrecq** [comment]
Deploy preview for *project-koku* ready!

Built with commit 5ee603c29815e1e26a0531191e2ed56dc714256e

https://deploy-preview-101--project-koku.netlify.com

File: src/pages/costDetails/detailsToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+interface DetailsToolbarOwnProps {
+  filterFields: any;
+  sortFields: any;
+  exportText: string;
+  onFiltersChanged?: PropTypes.func;
+  onSortChanged?: PropTypes.func;
+  onViewChanged?: PropTypes.func;
+  onActionPerformed?: PropTypes.func;
+  onFindAction?: PropTypes.func;
```
[the remark this answers is not in the document]
Comment (chambridge) [reply src/pages/costDetails/detailsToolbar.tsx]: I'll look more at the typing to see what can be specified for a function.

Yes, I'm planning on using calls to the API with the sorting and filtering features there.

**Comment by chambridge** [comment]
@dlabrecq I pushed changes for the PropTypes and to swap out FormControl with TextInput and PF3 Button for PF4 Button.

Styling isn't great for TextInput. I tried to set a `className` to try and override the lineHeight and fontSize but when I refreshed it still seemed to take from textInput.styles.ts maybe you or @boaz1337 know the quick fix for this, but I figured it doesn't have to be a blocker.

**Comment by chambridge** [comment]
Fixed the styling issues by instead styling some of the PF3 items to match PF4 better:

image

**Pull request merged**